components/textbook-beta/ExternalRecommendedReadings.vue

```diff
@@ -0,0 +1,36 @@
+<template>
```

Comment:
Looks like this is handled in a new [PR](https://github.com/Qiskit/qiskit.org/pull/2566)